src/main/java/com/puppycrawl/tools/checkstyle/utils/TokenUtil.java

```diff
@@ -281,7 +281,14 @@ public static boolean isTypeDeclaration(int type) {
      * @return true if type matches one of the given types.
      */
     public static boolean isOfType(int type, int... types) {
-        return Arrays.stream(types).anyMatch(tokenType -> tokenType == type);
+        boolean matching = false;
+        for (int t : types) {
```

Comment:
No single char names, name it "type"

Reply:
https://github.com/checkstyle/checkstyle/blob/c1c41d49ca6829df3c0dcf3e8ca980838c8ef0bf/config/checkstyle_checks.xml#L758-L761

Reply:
https://github.com/checkstyle/checkstyle/blob/c1c41d49ca6829df3c0dcf3e8ca980838c8ef0bf/config/checkstyle_checks.xml#L758-L761

Reply:
It probably makes sense to make it false. We like exact names.

Reply:
According to the check documentation, this would also apply to traditional for-loop. I do not think that we should enforce this unless we make property false and suppress warnings for traditional for-loops.

Reply:
https://github.com/checkstyle/checkstyle/pull/11521

Reply:
Changed to `foundType`, as `type` was already taken by the `type` to find a match for (in the method arguments).

Reply:
Please use tokenType as it was before